public/components/navigation/navigation.stache

```diff
@@ -3,22 +3,22 @@
 		<a href="/" class="pull-left main-logo">BitBalls</a>
 		<ul class="main-menu nav navbar-nav">
 		  <li role="presentation" {{#routeCurrent page='tournaments'}}class='active'{{/routeCurrent}}>
-				<a href="{{routeUrl page='tournaments'}}">Tournaments</a>
+				<a href="{{routeUrl (page='tournaments')}}">Tournaments</a>
```

Comment:
Does the space visually show it's a `helper`, or is that just assumed from being in the double curly braces?  I recall Justin's presentation showed it this way with the space, but I could not tell you why.